MicrosoftFluentUI.podspec

```diff
@@ -41,6 +42,7 @@ Pod::Spec.new do |s|
   s.subspec 'BarButtonItems_ios' do |barbuttonitems_ios|
     barbuttonitems_ios.platform = :ios
     barbuttonitems_ios.dependency 'MicrosoftFluentUI/Core_ios'
+    barbuttonitems_ios.preserve_paths = ["ios/FluentUI/BarButtonItems/BarButtonItems.resources.xcfilelist"]
```

Comment:
If I need to add a resource for Button then is there anything I need to do differently as a part of my workflow?

Reply:
You need to add the path to the new resource (relative to the root of the FluentUI-ios.xcassets) to make sure it won't be removed if the client app is using only the Button subspec.